client/main.lua

```diff
@@ -295,45 +305,63 @@ RegisterNetEvent("qb-garages:client:VehicleList", function(data)
-                                type = type,
-                                garage = garage,
-                                index = indexgarage,
-                            }
+                        title = Lang:t('menu.header.depot', {
+                            value = vname,
+                            value2 = v.depotprice
+                        }),
+                        metadata = {
+                            {label = 'Plate', value = v.plate},
+                            {label = 'Fuel', value = currentFuel .. "%"},
+                            {label = 'Engine', value = enginePercent .. "%"},
+                            {label = 'Body', value = bodyPercent .. "%"}
+                        },
+                        event = "qb-garages:client:TakeOutDepot",
+                        args = {
+                            vehicle = v,
+                            type = type,
+                            garage = garage,
+                            index = indexgarage
                         }
                     }
                 else
                     MenuGarageOptions[#MenuGarageOptions + 1] = {
-                        header = Lang:t('menu.header.garage', { value = vname, value2 = v.plate }),
-                        txt = Lang:t('menu.text.garage', { value = v.state, value2 = currentFuel, value3 = enginePercent, value4 = bodyPercent }),
-                        params = {
-                            event = "qb-garages:client:takeOutGarage",
-                            args = {
-                                vehicle = v,
-                                type = type,
-                                garage = garage,
-                                index = indexgarage,
-                            }
+                        title = Lang:t('menu.header.garage', {
+                            value = vname,
+                            value2 = v.plate
+                        }),
+                        metadata = {
+                            {label = 'State', value = v.state},
+                            {label = 'Fuel', value = currentFuel .. "%"},
+                            {label = 'Engine', value = enginePercent .. "%"},
+                            {label = 'Body', value = bodyPercent .. "%"}
+                        },
+                        event = "qb-garages:client:takeOutGarage",
+                        args = {
+                            vehicle = v,
+                            type = type,
+                            garage = garage,
+                            index = indexgarage
                         }
                     }
                 end
             end
 
             MenuGarageOptions[#MenuGarageOptions + 1] = {
-                header = leave,
-                txt = "",
-                params = {
-                    event = "qb-menu:closeMenu",
-                }
+                title = leave,
+                onSelect = function(_)
```

Comment:
```suggestion
                onSelect = function()
```
The `_` isn't needed in functions where no parameters are used at all.